finagle-redis/src/main/scala/com/twitter/finagle/redis/SubscribeClient.scala

```diff
@@ -0,0 +1,108 @@
+package com.twitter.finagle.redis
+
+import com.twitter.finagle.Service
+import com.twitter.finagle.redis.protocol._
+import com.twitter.finagle.redis.util.StringToChannelBuffer
+import org.jboss.netty.buffer.ChannelBuffer
+import com.twitter.util.Future
+import com.twitter.finagle.ServiceFactory
+import com.twitter.finagle.redis.util.CBToString
+import com.twitter.util.Futures
+import com.twitter.finagle.util.DefaultTimer
+import com.twitter.conversions.time._
+import com.twitter.util.Time
+import com.twitter.util.TimerTask
+import com.twitter.finagle.ServiceClosedException
+
+object SubscribeClient {
+
+  type MessageHandler = (ChannelBuffer, ChannelBuffer) => Unit
+  type PMessageHandler = (ChannelBuffer, ChannelBuffer, ChannelBuffer) => Unit
+
+  /**
+   * Construct a sentinel client from a single host.
+   * @param host a String of host:port combination.
+   */
+  def apply(host: String): SubscribeClient = {
+    new SubscribeClient(com.twitter.finagle.Redis.Subscribe.newService(host))
+  }
+
+  object MessageBytes {
+    val SUBSCRIBE = StringToChannelBuffer("subscribe")
```

Comment:
Upper case is used in CommandBytes, so I followed the same style here. MessageBytes is used only in SubscribeClient, so I made it private, and it would be safe to rename them later.

BTW: Sometimes, I find it hard to decide the case the name of a val should be in. What are the conventions are you follow in twitter?